gef-ui/server/utils/helpers.js

```diff
@@ -23,6 +23,9 @@ const userToken = (req) => {
 // Checks to see if an element is an object or not
 const isObject = (el) => typeof el === 'object' && el !== null && !(el instanceof Array);
 
+const isMIAWithoutChangedToIssuedFacilities = (app) =>
+  app.status === CONSTANTS.DEAL_STATUS.CHANGES_REQUIRED && app.submissionType === CONSTANTS.DEAL_SUBMISSION_TYPE.MIA && app.submissionCount > 0;
```

Comment:
Good practice to have conditions in new line, to improve readability.

Reply:
done